bitcoin/src/Bitcoin/Transaction/Taproot.hs

```diff
@@ -145,34 +121,34 @@ leafHash leafVersion leafScript =
 
 -- | Representation of a full taproot output.
 data TaprootOutput = TaprootOutput
-    { taprootInternalKey :: PubKey
+    { taprootInternalKey :: PubKeyXO
     , taprootMAST :: Maybe MAST
     }
     deriving (Show)
 
 
-taprootOutputKey :: TaprootOutput -> PubKey
+taprootOutputKey :: TaprootOutput -> PubKeyXY
```

Comment:
We should change this to `PubKeyXO` as well.

Reply:
Ah good catch, I thought it outputted a full parity-aware public key but upon reviewing the [spec](https://github.com/bitcoin/bips/blob/master/bip-0341.mediawiki#script-validation-rules) it appears that it is only 32 bytes.